README.md

```diff
@@ -34,6 +34,9 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 
 The project uses the APIs from Trello.
 To get this app to work for you, you will need to have a Trello account and store the following variables in your .env file:
-  API_KEY  -  Your unique key for using the Trello API
-  API_TOKEN  -  Generate your own API token and store value here
-  BOARD_ID  -  The ID of the board you want to use for the project
+```bash
+  * API_KEY  -  Your unique key for using the Trello API
+  * API_TOKEN  -  Generate your own API token and store value here
+  * BOARD_ID  -  The ID of the board you want to use for the project
+```
+Use the .env.template to create your own entries
```

Comment:
You should also add a section mentioning the tests, how to run them, and prerequisites like chromedriver.exe